### ISSUE TITLE
Uncaught Type Error

### ISSUE THREAD

**Issue by maeonian** [zionbuilder/zionbuilder#548]
**Describe the bug**
Since the latest update I'm unable to edit any Zion templates. I get a white screen, and the following error when I check the console logs:

```
Uncaught (in promise) TypeError: o.value is null
    setup https://********.com/wp-content/plugins/zionbuilder-pro/dist/js/theme-builder.js?ver=2.6.1:1
```

**Have you tried all the steps at https://zionbuilder.io/documentation/having-problems-using-zion-builder/?**
Yes

**Steps to reproduce**
1. Try to edit / create a template

**Expected behavior**
*What should have happened after following the steps above.*
Able to edit templates with Zion builder

**What actually happens**
*Provide a concise message of what is actually happening.*
A white / blank pages shows instead of any Zion editor features/panels

**Screenshots/Videos**
*If applicable, add screenshots/videos to help explain the problem.*

**Environment (please complete the following information):**
- OS: Windows 10
- Browser Firefox 89.0.2
- Zionbuilder Version 2.6.1 / pro 2.6.1

**Additional context**
*Add any other context about the problem here.*

**Comment by dajy** [comment]
Hi maeonian, we apologise for the late answer. Since we've implemented the feedback and roadmap platform on our website we didn't check github as often as before. 

This already has been fixed for the 2.6.2 update

You can edit templates from the left if you want . I see that only from the right it doesn't work, but if you add the component in a template then you can edit it.

**Comment by zauan** [comment]
Closing due to inactivity. Please feel free to reopen the ticket if you need further assistance.
